components/expenses/ExpenseAmountBreakdown.js

```diff
@@ -41,7 +41,8 @@ const ExpenseAmountBreakdown = ({ items, currency, taxes, expenseTotalAmount })
         <Flex flexDirection="column" alignItems="flex-end">
           <AmountLine data-cy="expense-invoiced-amount">
             <Span textTransform="capitalize" mr={3}>
-              <FormattedMessage defaultMessage="Subtotal" /> ({currency})
+              <FormattedMessage defaultMessage="Subtotal" />
+              {currency && ` (${currency})`}
```

Comment:
To prevent displaying empty parenthesis when the amount is not set. Same for the change lines 59-60.